.github/workflows/build-and-test-x86.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           # release build without optimized assembly routines
           {name: "release", flags: "--release --no-default-features --features=bitdepth_8,bitdepth_16", timeout_multiplier: 1},
           # debug build to catch overflows with optimized assembly routines
-          {name: "debug", flags: "", timeout_multiplier: 2}
+          {name: "opt-dev", flags: "--profile opt-dev", timeout_multiplier: 2}
```

Comment:
I think we should just have `profile` (`release`, `dev`, `opt-dev`) and `profile_dir` (`release`, `debug`, `opt-dev`) variables here.  Then we use `profile` for `--profile ${{ matrix.build.profile }}` and `profile_dir` for `target/${{ matrix.target }}/${{ matrix.build.profile_dir }}`.  It's very annoying that `debug`/`dev` have different names and directories unlike all the other ones.

Reply:
Oh I wrote this before you update `name` from `debug` to `opt-dev`, but it might clearer this way.